starter-repos/app-template/config/envs/prod-central/deployment.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Deployment
```

Comment:
Do we also need a `envs/dev` folder for running `skaffold`?

Reply:
It's also okay if we want to add this later along with the `skaffold.yaml` file. 
cc @yuwenma

Reply:
Yes, we should add one. I'll add it.

Reply:
my fault, I have the PR about adding the dev for skaffold. I'll upload mine after this is merged.